lib/api/workloadendpoint.go

```diff
@@ -60,10 +63,10 @@ type WorkloadEndpointSpec struct {
 	// internal IP will not have their source address changed, except when an endpoint attempts
 	// to connect one of its own external IPs. Each internal IP must be associated with the same
 	// endpoint via the configured IPNetworks.
-	IPNATs []IPNAT `json:"ipNAT,omitempty" validate:"omitempty,dive"`
+	IPNATs []IPNAT `json:"ipNATs,omitempty" validate:"omitempty,dive"`
```

Comment:
Can we have validation in place to ensure that any specified IPNATs match the specified IPNetworks?